--- /usr/local/lib/python3.8/dist-packages/pix2tex/cli.py	2024-10-23 19:35:59.000000000 +0900
+++ cli.py	2024-10-23 19:46:37.053516037 +0900
@@ -44,7 +44,8 @@
         ratios = [a/b for a, b in zip(img.size, max_dimensions)]
         if any([r > 1 for r in ratios]):
             size = np.array(img.size)//max(ratios)
-            img = img.resize(size.astype(int), Image.BILINEAR)
+            size = tuple(size.astype(int))
+            img = img.resize(size, Image.BILINEAR)
     if min_dimensions is not None:
         # hypothesis: there is a dim in img smaller than min_dimensions, and return a proper dim >= min_dimensions
         padded_size = [max(img_dim, min_dim) for img_dim, min_dim in zip(img.size, min_dimensions)]
